lib/aiken/cbor.ak

```diff
@@ -236,7 +236,7 @@ fn encode_base16(bytes: ByteArray, ix: Int, builder: ByteArray) -> ByteArray {
   }
 }
 
-fn from_int(i: Int, digits: ByteArray) -> ByteArray {
+pub fn from_int(i: Int, digits: ByteArray) -> ByteArray {
```

Comment:
This function isn't meant to be exposed as it has nothing to do in the API of aiken/cbor.

There's a small added cost coming from using cbor.diagnostic in the string module but surely it doesn't matter, right?